src/cljs/commiteth/manage_payouts.cljs

```diff
@@ -57,6 +57,19 @@
                        (:claims bounty))]
             [claim-card bounty claim]))))
 
+(defn bounty-stats [{:keys [paid unpaid]}]
+  (let [sum-dollars (fn [bounties]
+                      (reduce + (map #(js/parseFloat (:value_usd %)) bounties)))]
+    [:div.cf.pv4
+     [:div.fl.w-50.tc
+      [:div.ttu.tracked "Open"]
+      [:div.f2.pa2 "$" (sum-dollars unpaid)]
+      [:div (count unpaid) " bounties"]]
+
+     [:div.fl.w-50.tc
+      [:div.ttu.tracked "Paid"]
+      [:div.f2.pa2 "$" (sum-dollars paid)]
+      [:div (count paid) " bounties"]]]))
```

Comment:
I'll try to explain the different classes used here: 
- `.cf` = **c**lear**f**ix
- `.pv4` = **p**adding **v**ertical step 4 (see [scales](https://tachyons-tldr.now.sh/#/scales))
- `.fl` = **f**loat **l**eft
- `.w-50` = **w**idth **50**%
- `.tc` = **t**ext-align **c**enter
- `.ttu`= **t**ext-**t**ransform **u**ppercase
- `.tracked` = more spacing between letters
- `.f2` = **f**ont size step 2nd largest (see [scales](https://tachyons-tldr.now.sh/#/scales))
- `.pa2` = **p**adding **a**ll sides step 2

As you can see all classes do very specific things, extending them or otherwise modifying them would not be appropriate since they already are "perfect" according to spec.  Because they are so specific we can also easily compose them to create more complex components.

Using small immutable classes like this is really useful when working in a team since we no longer change (less specific) CSS classes (e.g. `.value`, `.ui`, `.label`) which different team members might have different understandings about at different times.

There's some pattern to those classnames which makes them quite easy to recall once you know them, until then this site is very useful to look up classes by name or what CSS properties they modify: https://tachyons-tldr.now.sh/